### DISCUSSION TITLE
Assigning roles to different users in different table

### DISCUSSION THREAD

**Discussion by smongare** [spatie/laravel-permission#2305] [Q&A]
Hi Everyone, 
I would like to know if I have two tables that store user information that have different columns. Can Spartie package allow assigning roles to those different users? If so, how will it be?
Thank you in advance.

**Comment by parallels999** [comment]
> Can Spartie package allow assigning roles to those different users?

Yes 

> If so, how will it be?

Read about multiple logins and multiple guards on laravel
